pkgs/top-level/all-packages.nix

```diff
@@ -31030,6 +31030,8 @@ with pkgs;
 
   webcamoid = libsForQt5.callPackage ../applications/video/webcamoid { };
 
+  webex = callPackage ../applications/networking/instant-messengers/webex.nix {};
```

Comment:
```suggestion
  webex = callPackage ../applications/networking/instant-messengers/webex {};
```
place the nix file at ...../webex/default.nix

Reply:
done